stripe/test/helper.py

```diff
@@ -135,6 +135,8 @@ def setUp(self):
             stripe.api_base = api_base
         stripe.api_key = os.environ.get(
             'STRIPE_API_KEY', 'tGN0bIwXnHdwOa85VABjPdSn8nWY7G7I')
+        stripe.api_version = os.environ.get(
+            'STRIPE_API_VERSION', '2017-02-14')
```

Comment:
I don't think we'll need to grab this out of the environment, but I do it anyway to reuse the pattern established for API keys.